test/framework/batch_helper.hpp

```diff
@@ -292,11 +297,33 @@ namespace framework {
      */
     template <typename... TxBuilders>
     auto makeTestBatchTransactions(TxBuilders &&... builders) {
-      auto reduced_hashes = internal::fetchReducedHashes(builders...);
-      auto transactions = internal::makeTxBatchCollection(
-          reduced_hashes, std::forward<TxBuilders>(builders)...);
+      return makeTestBatchTransactions(
+          shared_model::interface::types::BatchType::ATOMIC,
+          std::forward<TxBuilders>(builders)...);
+    }
+
+    /**
+     * Create test batch transactions from passed transaction builders with
+     * provided batch meta
+     * @tparam TxBuilders variadic types of tx builders
+     * @param batch_type type of the batch (either ATOMIC or ORDERED)
```

Comment:
Better to remove 

> (either ATOMIC or ORDERED)

Because in future it may be possible other different options. Thus, this comment will mislead readers.